src/ui/constants/common.ts

```diff
@@ -255,6 +255,7 @@ export const URL_OPTIONS = {
     EMBEDDED: '_embedded',
     NO_CONTROLS: '_no_controls',
     LANGUAGE: '_lang',
+    ACTION_PARAMS_ENABLED: '_ap_enabled',
```

Comment:
Bad name - dashkit will think that there is action param `enabled` for this chart